builder/builder-next/controller.go

```diff
@@ -107,7 +107,7 @@ func newController(rt http.RoundTripper, opt Opt) (*control.Controller, error) {
 		return nil, err
 	}
 
-	exec, err := newExecutor(root, opt.DefaultCgroupParent, opt.NetworkController)
+	exec, err := newExecutor(root, opt.DefaultCgroupParent, opt.NetworkController, opt.Rootless)
```

Comment:
With more arguments being added, should we just pass `opt` here? (`newExecutor` is not exported, so ok to discuss for a follow-up)

Reply:
Let's leave this to follow-up